site/examples/miscellaneous.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const tooltips = `<div class='flex-parent flex-parent--center-main flex-parent--wrap flex-parent-inline'>
+  <div class='bg-darken75 color-white align-center col col--12 p6 round txt-bold txt-s'>Hello world!</div>
+  <span class='triangle triangle--d'></span>
+</div>`;
+
+const simpleBadge = `<div class='bg-blue-faint color-blue inline-block px6 py3 txt-xs txt-bold round-full'>
+  screamin dealz
+</div>`;
+
+const tipBadge = `<div class='bg-green-faint color-green inline-block px6 py3 txt-xs txt-bold round-full'>
+  <svg class='icon icon--s inline-block align-middle'><use xlink:href='#icon-info'></use></svg>
+  Did you know?
+</div>`;
+
+const warningBadge = `<div class='bg-orange-faint color-orange-dark inline-block px6 py3 txt-xs txt-bold round-full'>
+  Heads up!
+  <button>
+    <svg class='icon icon--s inline-block align-middle ml3'><use xlink:href='#icon-close'></use></svg>
+  </button>
+</div>`;
+
+const progressBar = `<div class='h12 bg-darken10 relative round-full'>
+  <div class='absolute h12 bg-green-light round-full' style='width:50%;'></div>
+</div>`;
+
+const chip = `<div class='flex-parent flex-parent--center-cross flex-parent-inline relative w240'>
```

Comment:
This isn't a chip :P I'm happy to remove it. @samanpwbb @dasulit @davidtheclark is there a better name for this?

<img width="322" alt="screen shot 2017-01-17 at 2 02 03 pm" src="https://cloud.githubusercontent.com/assets/61150/22035333/8e424346-dcbd-11e6-8e2f-a293141109ab.png">